purchase_supplier_rank/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module updates supplier_rank for partners when creating purchase orders.
```

Comment:
Maybe add a comment here that normally supplier_rank is only updated by creating vendor invoices.

Reply:
Please @amarcosg check this! Thank you!

Reply:
Description updated, thanks for the review!